comps/finetuning/handlers.py

```diff
@@ -62,18 +60,12 @@ def handle_create_finetuning_jobs(request: FineTuningJobsRequest, background_tas
     train_file = request.training_file
     train_file_path = os.path.join(DATASET_BASE_PATH, train_file)
 
-    model_config_file = MODEL_CONFIG_FILE_MAP.get(base_model)
-    if not model_config_file:
-        raise HTTPException(status_code=404, detail=f"Base model '{base_model}' not supported!")
-
     if not os.path.exists(train_file_path):
         raise HTTPException(status_code=404, detail=f"Training file '{train_file}' not found!")
 
-    with open(model_config_file) as f:
-        finetune_config = parse_yaml_raw_as(FinetuneConfig, f)
```

Comment:
It would lose many flexibility by not using yaml. 
For example, how do you use lora? looks like lora is not used by default. 
Also, finetuning of rerank and embedding model is somewhat different with the default config for instruction tuning, thus need pass in some parameters different with the default ones, how to handle such case? parameters of `FineTuningJobsRequest` seems not enough to cover most circumstances.

Reply:
1. using yaml doesn't have flexibility for different models, so I remove them.

2. the lora setting is same with yaml (I change `lora_config: Optional[LoraConfig] = None` to lora_config: Optional[LoraConfig] = LoraConfig()`)

4. I will extend FineTuningJobsRequest to be able to customize `GeneralConfig`, `DatasetConfig`, `TrainingConfig`

Reply:
Using yaml in current way of binding yaml to model indeed does not have flexibility, but when passing yaml to FineTuningJobsRequest, I think it will have much flexibility.

Reply:
so we should extend FineTuningJobsRequest to be able to customize GeneralConfig, DatasetConfig, TrainingConfig? do you agree?

Reply:
Yes, either add parameters directly in FineTuningJobsRequest or just add an argument for yaml.

Reply:
ok, I will add parameters directly in FineTuningJobsRequest